### PR TITLE
fix: meta title about us

### LINKED ISSUE #28: Title page show `Senarai | [object Object]`
## Description

The page titles on the "Tentang Kami" page show `Senarai | [Object Object]`

## Expected Behaviour

The title should be `Senarai | Tentang Kami` (I guess)

## Additional Information

Screen Shot 2022-09-24 at 11 26 16

### PR DESCRIPTION
**Pull request by iamyuu**
Closes #28 

#### before 

Screen Shot 2022-09-30 at 08 12 50

#### after

Screen Shot 2022-09-30 at 08 13 15

### CODE REVIEW

**Comment by iamyuu** [comment]
after re-check, I don't know somehow the title is fixed

Screen Shot 2022-10-02 at 08 10 48

but on my local main branch, the issues still persist

Screen Shot 2022-10-02 at 08 11 43

🤷‍♂️

**Comment by zainfathoni** [comment]
> after re-check, I don't know somehow the title is fixed
> 
> Screen Shot 2022-10-02 at 08 10 48
> 
> but on my local main branch, the issues still persist
> 
> Screen Shot 2022-10-02 at 08 11 43
> 
> 🤷‍♂️

This repository is only deployed to https://senarai.vercel.app.
The https://senar.ai website is still using the deno instance here https://github.com/senar-ai/web 😅 
So yes, your fix should be valid.

File: app/routes/tentang-kami.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const handle: Handle = { name: 'Tentang Kami' }
 
 export const meta: MetaFunction = () => {
   return {
-    title: `Senarai | ${handle}`,
+    title: `Senarai | ${handle.name}`,
```
Comment (zainfathoni) [inline app/routes/tentang-kami.tsx:12]: While you're at it, can you please add a new test (either unit test or e2e test) to cover this particular use case by asserting the title tag?
Thanks! 🙏

**Review by zainfathoni — CHANGES_REQUESTED** [review]
LGTM 💯 
But let's prevent such regressions in the future by adding more tests. 😉 
Thanks!

File: app/routes/tentang-kami.tsx (hunk 1 above)
Comment (iamyuu) [reply app/routes/tentang-kami.tsx:12]: Sure! let me try that

**Review by zainfathoni — APPROVED** [review]
LGTM 💯 
Thanks for writing the test!

**Comment by zainfathoni** [comment]
@all-contributors please add @iamyuu for code and test.

**Pull request merged**